Package_SON.pretty/VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm.kicad_mod

```diff
@@ -0,0 +1,44 @@
+(module VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (layer F.Cu) (tedit 5B9F93A0)
+  (descr http://rohmfs.rohm.com/en/techdata_basic/ic/package/vson010v3030_1-e.pdf)
+  (tags "VSON-10 3x3mm")
+  (attr smd)
+  (fp_text reference REF** (at -0.006 2.572) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (at 0.248 -2.635) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -1.5 -1.5) (end -1.5 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -1.5 1.5) (end 1.5 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 1.5 1.5) (end 1.5 -1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 1.5 -1.5) (end -1.5 -1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -1.6764 -1.6764) (end -1.6764 1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.6764 1.6764) (end 1.6764 1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.6764 1.6764) (end 1.6764 -1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.6764 -1.6764) (end -1.6764 -1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start -0.8636 -1.8796) (end -1.8288 -1.8796) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.8796 -1.8796) (end -1.8796 -0.9652) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.78 -1.78) (end -1.78 1.78) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.78 1.78) (end 1.78 1.78) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.78 1.78) (end 1.78 -1.78) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.78 -1.78) (end -1.78 -1.78) (layer F.CrtYd) (width 0.05))
```

Comment:
The courtyard should provide mechanical **and electrical** clearance. Currently, the courtyard is not accounting for the copper pads on the sides of the part. The apparent clearance of 0.25mm from the body of the component should be used for the copper pads as well.